VSTS/Tests/lib/vsts-task-lib/mock.ts

```diff
@@ -1,44 +0,0 @@
-/// <reference path="../../../definitions/node.d.ts" />
-
-import path = require('path');
```

Comment:
You won't need any of the typescript stuff. VSTS tasks started with PowerShell but then quickly moved to nodejs because the build team wanted cross plat. So the entire repo uses nodejs related technologies. But since you are only looking after powershell tasks, you won't need this .ts stuff.